### ISSUE TITLE
Opening a large file breaks language features even if documentSymbols are disabled

### ISSUE THREAD

**Issue by xorye** [eclipse-lemminx/lemminx#635]
Steps to reproduce the issue:
1. Download the XSD file located here: https://github.com/redhat-developer/vscode-xml/issues/218#issue-547209289. This is a large file with over 120000 lines.
2. Place the file in a folder.
3. Open VS Code and make sure that `xml.symbol.enable` is set to `false`.
4. Open the folder.
5. Create new XML in the opened folder.
6. Open the large XSD file.
7. Open the XML files, language features like completion and validation are not working.

Gif of the problem:
![gif](https://raw.githubusercontent.com/xorye/gifs/master/new_issues/xml/whathappens.gif?token=AE3CR5IFJQEIYEJ2JEH4VRS6T5EQY)

**Comment by angelozerr** [comment]
I cannot reproduce it -( @fbricon can you reproduce it?

**Comment by fbricon** [comment]
Yup same issue. I think the problem is vscode sending the 7.9MB content to the XML server. It's unclear whether the server receives anything. We might need to do some profiling/tracing on the client side. Also worth trying, setting a large Xmx value (1GB) in the xml server vmargs

**Comment by fbricon** [comment]
Yeah increasing the heap really helps: "xml.server.vmargs": "-noverify -Xmx1G -XX:+UseG1GC -XX:+UseStringDeduplication"

**Comment by angelozerr** [comment]
Memory problem for this XSD file was fixed.
